src/plugins/widgets/search/SearchSettings.tsx

```diff
@@ -32,6 +34,43 @@ const SearchSettings: React.StatelessComponent<Props> = ({
         )}
       </select>
     </label>
+
+    <label>
+      <input
+        type="checkbox"
+        checked={active}
+        onChange={() => onChange({ active: !active })}
+      />
+      {' '}
+      Suggestions
+    </label>
+
+    {
+      active ?
+        <label>
+          Quantity
+          <input
+            type="number"
+            value={quantity}
+            onChange={event => {
+                let newQuantity = Number(event.target.value);
+
+                if (newQuantity < 1) {
```

Comment:
You could probably get away with just `min` and `max` props set on the `<input />` element.